python/cuml/dask/common/dask_df_utils.py

```diff
@@ -17,14 +17,19 @@
 from dask.distributed import default_client
 from toolz import first
 import dask.dataframe as dd
+from collections import OrderedDict
+
+from collections import OrderedDict
```

Comment:
double import?

Reply:
I just fixed and pushed.

Reply:
We both made the same changes and then I merged with yours. I suppose since they were on two different lines, git didn't flag it as a conflict.